spec.md

```diff
@@ -593,10 +593,11 @@ An identifier. Note that an identifier may be an expression or a destructuring p
 interface Literal <: Node, Expression {
     type: "Literal";
     value: string | boolean | null | number | RegExp;
+    kind: "string" | "boolean" | "null" | "number" | "regexp";
```

Comment:
27d0e173d7b220044c5fa05da1e9c3cfcf96a41b already established a precedent of referring to regular expressions with `regex`, which we should maintain for internal consistency.